CONTRIBUTING.md

```diff
@@ -87,69 +95,149 @@ You may need to install the dev_requirements for this
+| `azext_dt_region`	| Digital Twin Tests	| The region to restrict Digital Twins creation. If not specified, will create the Digital Twins in a supported region.	|
+| `azext_dt_adx_cluster`	| Digital Twin Data History Tests	| The name of the Azure Data Explorer Cluster to use. Azure Data Explorer Cluster creation time can take up to 20 minutes, so having a cluster is recommended.	|
+| `azext_dt_adx_database`	| Digital Twin Data History Tests	| The name of the database in the Azure Data Explorer Cluster to use.	|
+| `azext_dt_adx_rg`	| Digital Twin Data History Tests	| The resource group that contains the Azure Data Explorer Cluster. If not provided, `az_iot_testrg` will be used as the resource group. 	|
+| `azext_dt_ep_eventgrid_topic`	| Digital Twin Endpoint Tests	| The Event Grid Topic to use.	|
+| `azext_dt_ep_servicebus_namespace`	| Digital Twin Endpoint Tests	| The Service Bus Namespace to use.	|
+| `azext_dt_ep_servicebus_policy`	| Digital Twin Endpoint Tests	| The policy for the topic in the Service Bus Namespace to use.	|
+| `azext_dt_ep_servicebus_topic`	| Digital Twin Endpoint Tests	| The topic in the Service Bus Namespace to use.	|
+| `azext_dt_ep_eventhub_namespace`	| Digital Twin Data History and Endpoint Tests	| The Event Hub Namespace to use.	|
+| `azext_dt_ep_eventhub_policy`	| Digital Twin Endpoint Tests	| The policy for the Event Hub instance in the Event Hub Namespace to use.	|
+| `azext_dt_ep_eventhub_topic`	| Digital Twin Data History and Endpoint Tests	| The Event Hub instance in the Event Hub Namespace to use.	|
+| `azext_dt_ep_eventhub_topic_consumer_group`	| Digital Twin Data History Tests	| The Event Hub consumer group name to use. Defaults to "$Default".	|
+| `azext_dt_ep_rg`	| Digital Twin Data History and Endpoint Tests	| The resource group that contains the endpoint (Event Hub, Event Grid, Service Bus) variables. If not provided, `az_iot_testrg` will be used as the resource group.	|
 
-`azext_iot_teststoragecontainer` is the name of blob container belonging to the above mentioned storage account. Optional environment variable, defaults to 'devices' when not specified.
 
-##### IoT Hub
+##### IoT Central
 
-Execute the following command to run the IoT Hub integration tests:
+IoT Central integration tests can be run against a specific available api versions using command line argument "--api-version"
 
-`pytest azext_iot/tests/iothub/ -k "_int"`
+e.g. run tests against v 1.0
 
-##### Device Provisioning Service
+`pytest azext_iot/tests/central/ -k "_int.py" --api-version "1.0"`
 
-Execute the following command to run the IoT Hub DPS integration tests:
+If the "--api-version" argument is not specified, all runs act against default api version for each tested command.
 
-`pytest azext_iot/tests/dps/ -k "_int"`
+##### Test Resource Tagging
 
-##### IoT Central
-Execute the following command to run the IoT Central integration tests:
+There are 4 more test variables used for tagging the test resources:
```

Comment:
Not including these test variables in the IT variables table or list above?

Reply:
I am trying to show that these are separate and would be used by internal developers (as in us)

Reply:
updated